### ISSUE TITLE
Some games don't seem to sync, nor playtime

### ISSUE THREAD

**Issue by profucius** [XenorPLxx/playnite-library-psn#38]
It seems that some games/playtime are not syncing from Playstation via your plugin into Playnite.

I noticed that some games I played within the past year on my Sony PS5 (Assassin's Creed Mirage, Stellar Blade, Astro Bot) were not added to my Playnite on syncing via your addon. I wanted to see if the addon is working at all, so I deleted a different game (God of War Ragnarok, which I have no played since 2022) from Playnite, synced via your addon, and that game was re-added to my Playnite library. This tells me that the addon is syncing, but for some reason those games above are not syncing at all.

Is there anything I can do to provide debug data to you?

**Comment by XenorPLxx** [comment]
Is there anything special about how you've played these games? Are these for example physical games that you only played when offline?

**Comment by profucius** [comment]
> Is there anything special about how you've played these games? Are these for example physical games that you only played when offline? Have you tried running one of these games recently and then syncing?

As far as I know, they are all Physical games. I don't think I've played any Digital games on my PS5. In my testing, I deleting God of War from Playnite, and when running an update in your addon, it re-downloaded God of War again. But Astro Bot still will not sync to Playnite at all, and it is a Physical game all the same.

My PS5 is online, which I can confirm because I use Remote Play to access it from another device in my home, I can reach the Play store, etc.

I've been playing Astro Bot a bit each day for the past week, and the addon still hasn't synced the game or playtime to my Playnite.

Is there any debug data I can collect?

**Comment by XenorPLxx** [comment]
PSN plugin imports and combines games from multiple sources, main ones are:
- PSN library page https://library.playstation.com/recently-purchased
- Mobile App API
- PSN played page https://library.playstation.com/recently-played
- Trophy API (only for PS3/PSP/PSVITA games)

Where I'd expect to find the games played from physical media is https://library.playstation.com/recently-played and maybe mobile app, so you can check if you can see them there. They will definitely NOT show up on recently purchased - that's just the digital ones.

There was already a discussion https://www.reddit.com/r/playnite/comments/199kzwf/playstation_games_not_imported_completely/?sort=new that resulted in a task to [add possibility of using trophies API for PS4/PS5 games](https://github.com/XenorPLxx/playnite-library-psn/issues/24), but it's not implemented yet - trophies API doesn't use the same IDs directly, so checking for duplicates will be harder, haven't had time to take a look at it yet.

**Comment by XenorPLxx** [comment]
Nope, thats a first ive heard of such thing. Reddit discussion was a bit different with pretty old games played years ago, but both should be solved when ill get time to figure out sync with trophy api for ps4/5.
